public/apps/configuration/panels/role-edit/test/role-edit-filtering.test.tsx

```diff
@@ -0,0 +1,975 @@
+          },
+          {
+            label: 'cluster:monitor/remote/info',
+          },
+          {
+            label: 'cluster:monitor/state',
+          },
+          {
+            label: 'cluster:monitor/stats',
+          },
+          {
+            label: 'cluster:monitor/task',
+          },
+          {
+            label: 'cluster:monitor/task/get',
+          },
+          {
+            label: 'cluster:monitor/tasks/list',
+          },
+          {
+            label: 'cluster:monitor/tasks/list*',
+          },
+          {
+            label: 'restapi:admin/actiongroups',
+          },
+          {
+            label: 'restapi:admin/allowlist',
+          },
+          {
+            label: 'restapi:admin/internalusers',
+          },
+          {
+            label: 'restapi:admin/nodesdn',
+          },
+          {
+            label: 'restapi:admin/roles',
+          },
+          {
+            label: 'restapi:admin/rolesmapping',
+          },
+          {
+            label: 'restapi:admin/ssl/certs/info',
+          },
+          {
+            label: 'restapi:admin/ssl/certs/reload',
+          },
+          {
+            label: 'restapi:admin/tenants',
+          },
+          {
+            label: 'indices:admin/template/delete',
+          },
+          {
+            label: 'indices:admin/template/get',
+          },
+          {
+            label: 'indices:admin/template/put',
+          },
+          {
+            label: 'indices:admin/index_template/delete',
+          },
+          {
+            label: 'indices:admin/index_template/get',
+          },
+          {
+            label: 'indices:admin/index_template/put',
+          },
+          {
+            label: 'indices:admin/index_template/simulate',
+          },
+          {
+            label: 'indices:admin/index_template/simulate_index',
+          },
+          {
+            label: 'indices:data/read/scroll',
+          },
+          {
+            label: 'indices:data/read/scroll/clear',
+          },
+          {
+            label: 'indices:data/write/bulk',
+          },
+          {
+            label: 'indices:data/write/bulk*',
+          },
+          {
+            label: 'indices:data/read/mget',
+          },
+          {
+            label: 'indices:data/read/mget*',
+          },
+          {
+            label: 'indices:data/read/msearch',
+          },
+          {
+            label: 'indices:data/read/msearch/template',
+          },
+          {
+            label: 'indices:data/read/mtv',
+          },
+          {
+            label: 'indices:data/read/mtv*',
+          },
+          {
+            label: 'indices:data/write/reindex',
+          },
+        ],
+      },
+    ]);
+  });
+
+  it('basic index permission panel rendering', async () => {
+    const action = 'create';
+    const buildBreadcrumbs = jest.fn();
+
+    (fetchActionGroups as jest.Mock).mockResolvedValue({
```

Comment:
If this is duplicated with the code above, would it be possible to write it once like is done here: https://github.com/opensearch-project/security-dashboards-plugin/pull/1503/files#diff-048f4ea5adf75b37a58ec2e3f1b3e0b3245c0f98e587ec0f8631a0422a7664deR21-R33?